addons/website_sale_referral/models/res_partner.py

```diff
@@ -0,0 +1,7 @@
+from odoo import api, fields, models
```

Comment:
api is imported but not used. And it seems to be the case at a lot of places.

Reply:
Should be better now